src/flow.h

```diff
@@ -374,6 +374,10 @@ typedef struct Flow_
 
     uint8_t vlan_idx;
 
+    struct {
+        uint32_t spi; /**< esp spi */
```

Comment:
Can we make this part of the `sp` union above?

Reply:
Interesting, when I add it to the union, it no longer tracks the flow?

Reply:
Did you update the FlowCompare func(s)?

Reply:
Yes, that was it, tricky unions... I wish it would give a warning at-least :(